### ISSUE TITLE
should we use html_date everywhere?

### ISSUE THREAD

**Issue by fuzzylogic2000** [liqd/a4-meinberlin#4017]
**URL:** 
**user:** 
**expected behaviour:** 
**behaviour:** We do use our own template tag `html_date` in quite a few places, but seem to have forgotten in others (like here https://github.com/liqd/a4-meinberlin/blob/fb7aaf29e24bc904383e2521399548d94c4bac24/meinberlin/apps/plans/templates/meinberlin_plans/plan_detail.html#L92). I think we need to decide on one format and use that.
**important screensize:**
**device & browser:** 
**Comment/Question:** The main question to answer is: is the `html_date` better for a11y, because it wraps the dates in a html time element? @phillimorland do you know? https://github.com/liqd/a4-meinberlin/blob/23922a3797db0e06b6de6cd1aa87bc73224cbc17/meinberlin/apps/contrib/templatetags/contrib_tags.py#L44

Screenshot?

**Comment by fuzzylogic2000** [comment]
@sabinammm s guess was that we might not need it anymore:
> It seems that the tag was created to make the date readable for screen readers. If this is no longer relevant because screen readers can now read the date without this extra parsing

**Comment by sabinammm** [comment]
In Browserstack tested with integrated screenreader for:

- Windows latest Firefox (94), Firefox (86), Chrome (96), Chrome (89) on Windows 11 and Windows 10. Older Windows version have no supported screen reader.
Reads all the text on the page and the date when the page is read in one go. This only doesn't work on Chrome 89: unless you hover over the text, 

- Safari 13.1 on macOS Catalina: If the text is selected, it is read. Tabbing through the page, only the links and title of the page are selected and read.

- Safari & Chrome on iPad Pro 12.9 2021 iOS, v14.6 2021: no screenreader

- Safari & Chrome on iPhone 12 Pro v14.4: no screen reader

**To sum up: it is legible by those screen readers tested without the use of html_date**

Question: Do we have a minimum standard yet that we use for accesssibility?

**Comment by phillimorland** [comment]
@sabinammm the a11y web standard is set by https://www.w3.org/WAI/ I would just check there for the element or if it's now deprecated, also here all html elements are listed here with their uses https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time seems like time might be a more searchn engine thing which might be useful
